### ISSUE TITLE
Question about multi rtsp streams

### ISSUE THREAD

**Issue by vishyVishal** [ultralytics/yolov5#8251]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

Hi. So I have 2 separately RTSP IP streams that I have added into a text file called 'streams.txt'. When I run it and stop it to look at the saved video. I have the video saved  which shows video from 1st item and then after a while from 2 item. But the 2nd video starts at a later time than the first video.
I want to confirm that the processing is done simultaneously for both streams? What would be the best way to confirm this?
In other words, while performing detections, how can I check that the source is which rtsp stream? is it by the thread ID or something?

Thank you for your time

**Comment by glenn-jocher** [comment]
@vishyVishal streams will show live in realtime if your environment supports cv2.imshow()

**Comment by vishyVishal** [comment]
Thanks Glenn for your reply. Is there a way in detect.py to know the detections in happening on which camera source? i want to make a POST request within the code and I want to send information of detection coming from which source.

thanks

**Comment by glenn-jocher** [comment]
@vishyVishal the batch is assembled sequentially according to the stream addresses in streams.txt, i.e. row zero is index zero in the batch results.
